jsapp/js/account/security/email/emailSection.component.tsx

```diff
@@ -87,6 +88,15 @@ export default function EmailSection() {
     });
   }
 
+  function handleClick() {
```

Comment:
Given there are more buttons in this component, I think `handleSubmit` would be more precise :)